src/backend/aspen/test_infra/models/usergroup.py

```diff
@@ -1,22 +1,31 @@
-from aspen.database.models import Group, User
+from aspen.database.models import Group, Location, User
 
 
 def group_factory(
     name="groupname",
     address="123 Main St",
     prefix=None,
-    location="Metropolis",
-    division="West",
+    location=None,
+    division=None,
 ) -> Group:
     # shortcut so we don't need to specify prefix
     if not prefix:
         prefix = name
+    # Note - the None checks are to allow explicitly empty location/division strings
+    if location == None:
+        location = f"{name} city"
+    if division == None:
+        division = f"{name} state"
+    tree_loc = Location(
+        region="North America", country="USA", location=location, division=division
+    )
     return Group(
         name=name,
         address=address,
         prefix=prefix,
         location=location,
         division=division,
+        default_tree_location=tree_loc,
```

Comment:
The testing scripts always throw me off. I was like, why did we do this twice but somewhat differently?? LOL